jax/util.py

```diff
@@ -88,6 +88,9 @@ def split_dict(dct, names):
 def concatenate(xs):
   return list(it.chain.from_iterable(xs))
 
+def consecutive_pairs(xs):
```

Comment:
IMO this should just be inlined rather than making a helper function. It's a common idiom, and it's already used all over (neural net code like mnist_classifier_fromscratch.py, library code like lax_numpy.py). Having to look up the one-liner over in util.py is actually harder than just seeing the definition inlined in the relevant code, and the descriptive name is not as good as the precise implementaiton.

Reply:
Agreed and done.